src/open_inwoner/cms/cases/views/status.py

```diff
@@ -213,6 +213,11 @@ def get_context_data(self, **kwargs):
                         "call_to_action_text",
                         None,
                     ),
+                    "case_link_text": getattr(
+                        self.statustype_config_mapping.get(end_statustype.url),
+                        "case_link_text",
+                        None,
```

Comment:
@stevenbal Do you think this is missing a translation? Not sure if this needs to be 'none' here... Should I add the default value here?

Reply:
I think this is missing a translation yes, `label` does use a translation for example: https://github.com/maykinmedia/open-inwoner/pull/962/files#diff-b85163f9f163b6780c498ce8ef19dd75145591086f469fa2b67cdfaec753a1e0R193

Reply:
Translation would only be needed for the label and the default value (?) since (on the fornt-end) this is a configurable text that will be changed by the admins - it works in the admin, but the label looks 'bold' on the Admin when I view it so not sure what is causing that.

Reply:
The label looks bold in the admin because the field is required right now

@alextreme we currently don't support translations for these admin configurable texts, right?

Reply:
@stevenbal admin and frontend are Dutch-only, if that is what you mean

Reply:
I meant frontend indeed :+1: